scripts/globals/spells/fira.lua

```diff
@@ -0,0 +1,29 @@
+-----------------------------------------
+-- Spell: Fira
+-- Deals fire damage to enemies within area of effect.
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/magic")
+-----------------------------------------
+
+function onMagicCastingCheck(caster, target, spell)
+    return 0
+end
+
+function onSpellCast(caster, target, spell)
+    local spellParams = {}
+    spellParams.hasMultipleTargetReduction = true
+    spellParams.resistBonus = 1.0
+    spellParams.V = 216
+    spellParams.V0 = 140
```

Comment:
bgwiki spell page lists V0 at 140, but the [magic damage page](https://www.bg-wiki.com/bg/Magic_Damage) lists it at 240.

V0 of 240 probably makes more sense when compared with V and V50~